cmake/FindGzOGRE2.cmake

```diff
@@ -280,6 +300,12 @@ if (PkgConfig_FOUND)
     set(GzOGRE2_VERSION_EXACT FALSE)
     set(GzOGRE2_VERSION_COMPATIBLE FALSE)
 
+    # check if ogre-next>=3.0.0 was found
+    check_possible_ogre_next_3_or_later(
+      OGRE2_VERSION_MAJOR
+      "OGRE-Next with major version ${OGRE2_VERSION_MAJOR} was found but OGRE-Next>=3.0.0 is not supported by GzOGRE2. Please, use instead GzOGRENext or specify a path to OGRE-Next<3.0.0 via an environment varialbe: PKG_CONFIG_PATH=<path-to-a-directory-containing-a-pkgconfig-file-for-OGRE-Next>."
```

Comment:
> or specify a path to OGRE-Next<3.0.0 via an environment varialbe: PKG_CONFIG_PATH=<path-to-a-directory-containing-a-pkgconfig-file-for-OGRE-Next>."

I think that we can skip this part about pkg-config if it is not related to CMake or gz-cmake.